src/components/editor/MonacoEditor.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="monaco-editor"
+    data-cy="monaco-editor"
+  >
+    <div id="container"></div>
+  </div>
+</template>
+
+<script setup>
+import {
+  onMounted,
+  onUpdated,
+  nextTick,
+} from 'vue';
+
+const monaco = require('monaco-editor');
+
+// Need to be here, otherwise OnUpdated is not call
+defineProps({
+  viewType: String,
+});
+
+let editor;
+
+/**
+ * Setup monaco editor.
+ */
+function createEditor() {
+  const container = document.getElementById('container');
```

Comment:
Should we mess with the DOM directly ? Is there a way to make this understood by Vue ?